packages/ui/src/elements/QueryPresets/fields/WhereField/index.tsx

```diff
@@ -104,7 +104,7 @@ export const QueryPresetsWhereField: JSONFieldClientComponent = ({
         {value
           ? transformWhereToNaturalLanguage(
               value as Where,
-              getTranslation(collectionConfig.labels.plural, i18n),
+              getTranslation(collectionConfig?.labels?.plural, i18n),
```

Comment:
We have other places in the code where we use `collectionConfig.labels.plural` without the optional chaining. I thought this was already sanitized.